dashboard/test/lib/policies/child_account_test.rb

```diff
@@ -36,7 +36,8 @@ class ComplianceStateTest < ActiveSupport::TestCase
       [[:non_compliant_child, :unknown_us_region], true],
       [[:non_compliant_child, :not_U13], true],
       [[:non_compliant_child, :migrated_imported_from_clever], true],
-      [[:non_compliant_child, :with_lti_auth], true],
+      [[:non_compliant_child, :with_lti_auth], false],
```

Comment:
This expected test result was changed to false because the changes to `personal_account?` revealed that it was actually incorrect. The test user is created with a password by default, which makes this a personal account, and therefore non-compliant because of the `:non-compliant_child` trait. I added another item to the matrix to test what the old one intended to test: a user a with an LTI auth option but no email login or password.